content/en/resources/news.md

```diff
@@ -1,9 +1,14 @@
 ---
 title: News
-description:
-  The [press](resources/press) page contains a full listing of news coverage
+description: TUF news coverage
+aliases: -[/news] -[/press]
```

Comment:
Please check the syntax reference for Hugo front matter: https://gohugo.io/content-management/front-matter/#aliases

Reply:
> Please check the syntax reference for Hugo front matter: https://gohugo.io/content-management/front-matter/#aliases

@lukpueh I followed that convention but the format checks do not allow me to do a multiline on the front matter. I'll wait to hear from @chalin to see how to go about it. Thanks for the reference.

Reply:
I've fixed this. It's standard YAML syntax, which can be either an inline list or a one-entry-per-line list.